packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -774,7 +774,7 @@ class _SelectableTextState extends State<SelectableText>
         autofocus: widget.autofocus,
         forceLine: false,
         minLines: widget.minLines,
-        maxLines: widget.maxLines ?? defaultTextStyle.maxLines,
+        maxLines: (widget.maxLines ?? defaultTextStyle.maxLines)?.toDouble(),
```

Comment:
I have not updated the maxLines property for `Text/SelectableText/DefaultTextStyle` hence we need to convert them to double before passing to the `EditableText`